### ISSUE TITLE
Unable to load a model (CPU) after upgrading to v0.10.0

### ISSUE THREAD

**Issue by ziqizhang** [MaartenGr/BERTopic#598]
So I was using 0.9.4 before without any problem until I realise that there was a bug in .visualize_topics(), that topic #0 is not included in the visual. Reading the issues on this repo, I understand I have to install v0.10.0 from the source code.

I believe I have done this properly, by doing
- pip uninstall bertopic
- python setup.py install

Then I tried to read a trained model (using v0.10.0) that has been converted to CPU, but I get this error:

Traceback (most recent call last):
  File "/home/zz/Programs/anaconda3/envs/bertopic/lib/python3.8/site-packages/bertopic-0.10.0-py3.8.egg/bertopic/_bertopic.py", line 1321, in load
  File "/home/zz/Programs/anaconda3/envs/bertopic/lib/python3.8/site-packages/joblib/numpy_pickle.py", line 577, in load
    obj = _unpickle(fobj)
  File "/home/zz/Programs/anaconda3/envs/bertopic/lib/python3.8/site-packages/joblib/numpy_pickle.py", line 506, in _unpickle
    obj = unpickler.load()
  File "/home/zz/Programs/anaconda3/envs/bertopic/lib/python3.8/pickle.py", line 1212, in load
    dispatch[key[0]](self)
  File "/home/zz/Programs/anaconda3/envs/bertopic/lib/python3.8/pickle.py", line 1589, in load_reduce
    stack[-1] = func(*args)
  File "/home/zz/Programs/anaconda3/envs/bertopic/lib/python3.8/site-packages/numba/core/serialize.py", line 97, in _unpickle__CustomPickled
    ctor, states = loads(serialized)
AttributeError: Can't get attribute '_rebuild_function' on <module 'numba.core.serialize' from '/home/zz/Programs/anaconda3/envs/bertopic/lib/python3.8/site-packages/numba/core/serialize.py'>

Can I get some help please? Thanks

**Comment by ziqizhang** [comment]
I have fixed this.

The reason is that there are some incompatible versions of packages in the environment I used to train the model, and the environment that I load the model. Specifically in my case, it is numba and scikit-learn

Because the model saving/loading uses pickle, and we don't know what objects are serialised, it is important to make sure the two environments (if they are not the same one!) must have identical packages installed.
